YunoHost/yunohost
# Update spectre meltdown checker

## The problem

Spectre Meltdown checker was last updated around May : https://github.com/YunoHost/yunohost/pull/464

## Solution

Update to most recent version with `git subtree pull --prefix src/yunohost/vendor/spectre-meltdown-checker https://github.com/speed47/spectre-meltdown-checker/ master --squash`

## PR Status

Tested, works as expected

## How to test

Pull the branch and run `yunohost tools diagnosis`, see if there's an output in `security` section

## Validation
